### PR TITLE
Add Allwinner H616 & Orange Pi Zero 2 support

### PR DESCRIPTION
**Pull request by ZhangGaoxing**
image

Benchmarking with Orange Pi Zero 2. The operating system is Armbian focal, Linux kernel version is 4.9.255, and .NET version is 6.0.1.

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/dotnet/iot/pull/1790)

### CODE REVIEW

File: src/devices/Gpio/Drivers/OrangePiZero2Driver.cs [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+{
+    /// <summary>
+    /// A GPIO driver for the Orange Pi Zero 2.
+    /// </summary>
+    /// <remarks>
+    /// SoC: Allwinner H616 (sun50iw9p1)
+    /// </remarks>
+    public class OrangePiZero2Driver : Sun50iw9p1Driver
+    {
+        public static readonly int[] _pinNumberConverter = new int[]
```
Comment (pgrawehr) [inline src/devices/Gpio/Drivers/OrangePiZero2Driver.cs:16]: This generates a "Missing XML Documentation" error. Probably it shouldn't be public?

**Review by krwq — APPROVED** [review]
Thanks @ZhangGaoxing! The benchmark result looks pretty awesome!

**Comment by krwq** [comment]
And sorry for late review! 👀

**Pull request merged**